src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed');
```

Comment:
I can't think of any case where `data-is-session-closed` will be undefined. we already stated in the backend that if the session is closed `data-is-session-closed = true` else `data-is-session-closed = false`. :sweat_smile: 
```java
Slots.IS_SESSION_CLOSED, sessionIsOpen ? "false" : "true"
```